### PR TITLE
refactor(*): update pvc quota name typo in comment

### PR DESCRIPTION
**Pull request by lsytj0413**
**What type of PR is this?**

/kind document

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/quota/v1/evaluator/core/persistent_volume_claims.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ func (p *pvcEvaluator) MatchingResources(items []corev1.ResourceName) []corev1.R
 			result = append(result, item)
 			continue
 		}
-		// match pvc resources scoped by storage class (<storage-class-name>.storage-class.kubernetes.io/<resource>)
+		// match pvc resources scoped by storage class (<storage-class-name>.storageclass.storage.k8s.io/<resource>)
```
Comment (neolit123) [inline pkg/quota/v1/evaluator/core/persistent_volume_claims.go:133]: this seems accurate.
deferring to the maintainers of this code for approval.

**Comment by lsytj0413** [comment]
/assign @deads2k
/assign @derekwaynecarr
/assign @vishh

PTAL

**Comment by lsytj0413** [comment]
@kubernetes/milestone-maintainers

**Pull request merged**